src/thunks/signInUser.js

```diff
@@ -0,0 +1,23 @@
+import { fetchData } from '../utils';
+import { setError, setLoading } from '../actions';
+import { getUserAndSchedule } from './getUserAndSchedule';
+import * as gql from '../queries';
+
+export const signInUser = (user, id, newUser = false) => {
```

Comment:
Great job reusing this, for both the signin and creatuser. We might want to consider refactoring this thunk and getUserAndSchedule thunk later, since they are both getting the user back as responses